farm/file_utils.py

```diff
@@ -388,10 +388,3 @@ def unnestConfig(config, flattened=False):
             unnestedConfig.append(tempconfig)
 
     return unnestedConfig
-
-
-def create_folder(dir):
-    if os.path.exists(dir):
-        logger.warning(f"Path {dir} already exists. You might be overwriting files.")
```

Comment:
I think it makes sense to at least sent a notification if you potentially overwrite files. 

The problem currently is that this code is called twice when saving a model (processor.save and adaptive_model.save). So this bit is always executed, even when saving to a new folder. We should check weather the folder contains files and then print the warning.

btw: os.makedirs(exist_ok) seems to be removed from Python 3.4.1 onwards. https://bugs.python.org/issue21082